common/scripts/kind_provisioner.sh

```diff
@@ -35,7 +35,8 @@ set -x
 DEFAULT_KIND_IMAGE="gcr.io/istio-testing/kind-node:v1.32.0"
 
 # the default kind cluster should be ipv4 if not otherwise specified
-IP_FAMILY="${IP_FAMILY:-ipv4}"
```

Comment:
Currently this includes kind_provisioner.sh changes too just for testing purpose and review. Once finalized the respective changes will move to istio/common-files repo